client/utils/auth.py

```diff
@@ -18,6 +18,8 @@
                                  create_config_directory)
 from client.utils.html import (auth_html, partial_course_html,
                                partial_nocourse_html, red_css)
+from client.utils import network
+from client.utils.sanction import Client
```

Comment:
This is no longer correct (see `from sanction import Client` above) (see master/#209)